File: questPresentation/src/main/java/ru/nekit/android/qls/quest/view/mediator/types/perimeter/PerimeterQuestViewHolder.java
package ru.nekit.android.qls.quest.view.mediator.types.perimeter;

import android.content.Context;
import android.support.annotation.NonNull;
import android.view.View;
import android.widget.TextView;

import ru.nekit.android.qls.R;
import ru.nekit.android.qls.utils.ViewHolder;

class PerimeterQuestViewHolder extends ViewHolder {

    TextView aFigureSideLabel, bFigureSideLabel;
    View figureView;

    PerimeterQuestViewHolder(@NonNull Context context) {
        super(context, R.layout.ql_perimeter);
        figureView = view.findViewById(R.id.view_figure);
        aFigureSideLabel = (TextView) view.findViewById(R.id.field_figure_side_a);
        bFigureSideLabel = (TextView) view.findViewById(R.id.field_figure_side_b);
    }

}

File: questPresentation/src/main/java/ru/nekit/android/qls/quest/view/mediator/types/perimeter/PerimeterQuestViewHolder.kt
package ru.nekit.android.qls.quest.view.mediator.types.perimeter

import android.content.Context
import android.view.View
import android.widget.TextView

import ru.nekit.android.qls.R
import ru.nekit.android.qls.utils.ViewHolder

//ver 1.0
internal class PerimeterQuestViewHolder(context: Context) : ViewHolder(context, R.layout.ql_perimeter) {

    var aFigureSideLabel: TextView = view.findViewById<View>(R.id.field_figure_side_a) as TextView
    var bFigureSideLabel: TextView = view.findViewById<View>(R.id.field_figure_side_b) as TextView
    var figureView: View = view.findViewById(R.id.view_figure)

}
